lntest/rpc/wallet_kit.go

```diff
@@ -254,6 +254,21 @@ func (h *HarnessRPC) BumpFee(
 	return resp
 }
 
+// BumpForceCloseFee makes a RPC call to the node's WalletKitClient and asserts.
+//
+//nolint:lll
```

Comment:
I actually don't think you need this `nolint` directive anymore afaict.

Reply:
removing the `nolint:lll` gave me an error for the `make lint` cmd tho ?

Reply:
disregard then. My GH render seemed to indicate otherwise, but nbd.